tests/test_models.py

```diff
@@ -253,13 +253,33 @@ async def test_model_update_save() -> None:
 
 async def test_model_bulk_update() -> None:
     await Movie(name="Boyhood", year=2004).insert()
+    await Movie(name="Boyhood-2", year=2011).insert()
 
-    movies = await Movie.query({Movie.year: 2004}).update({Movie.year: 2010})
+    movies = await Movie.query({Movie.year: 2004}).update(year=2010)
     assert movies[0].year == 2010
 
     movies = await Movie.query().all()
     assert movies[0].year == 2010
 
+    movies = await Movie.query({Movie.name: "Boyhood-2"}).update(year=2010)
+    assert len(movies) == 1
+    assert movies[0].year == 2010
+
+    movies = await Movie.query({Movie.year: 2010}).all()
+    assert len(movies) == 2
+
+    movies = await Movie.query({Movie.name: "Boyhood-2"}).update(
+        year=2014, name="Boyhood 2"
+    )
+    assert movies[0].year == 2014
+    assert movies[0].name == "Boyhood 2"
+
+    movies = await Movie.query({Movie.name: "Boyhood 2"}).update(
+        **{"year": 2010, "name": "Boyhood-2"}
```

Comment:
This is interesting. Shouldn't we accept this?

```python
await Movie.query().update({Movie.year: 2010})
```

Reply:
This is all very good. Thank you for the effort.

Reply:
This is not possible because of the way **kwargs works, it only allows the keys to be of type str and in this example the keys are of type ModelField.

Reply:
That makes sense. So let's update the docs examples. I'll leave comment there.

Reply:
I'd also remove this as this isn't really that much different from the other examples.